frontend/src/utils/fetch.tsx

```diff
@@ -9,7 +9,7 @@ export async function http<T>(request: RequestInfo): Promise<HttpResponse<T>> {
 
   try {
     res.parsedBody = await res.json()
-  } catch (ex) {}
+  } catch (ex) {} //eslint-disable-line
```

Comment:
I don't think ignoring this error is a good solution. Removing this try-catch and always parsing the body as json would solve all of the issues involving null checks on the parsed body. At the very least, log a message with the exception so we know when this happens whether the caller was expecting json or not. Or even better, don't assume the response is json and let the caller decide how they want to parse the body.